### DISCUSSION TITLE
Add mouse indication for tooltip presence for Godot Editor

### DISCUSSION THREAD

**Discussion by boukew99** [godotengine/godot-proposals#4147] [Editor]
It is not always clear what Controls have tooltips and there is also not always consistency on which you can rely, such as that only a few items in the MenuButtons have tooltips.  
A solution could be to set the cursor shape to CURSOR_HELP (normal mouse with small question mark)  if you have hover over a Control with a tooltip.

**Comment by Calinou** [comment]
This would likely be distracting in practice. Very few applications use `CURSOR_HELP` nowadays. It's generally reserved to clicking those Windows 95-style Help buttons in the window title bar buttons.

**Reply by boukew99** [reply]
Good point. I only tested it with few items, but with the editor inspector I can imagine it becoming too distracting.
